### ISSUE TITLE
Warnings about orchard dep shadowing clojure.core output when using Clojure 1.11.0

### ISSUE THREAD

**Issue by stevejmp** [liquidz/iced-nrepl#19]
## Expected Behavior
Use `iced repl` with no warnings in a project using Clojure 1.11.

## Actual Behavior
```
WARNING: update-vals already refers to: #'clojure.core/update-vals in namespace: mrandersonicednrepl.orchard.v0v7v1.orchard.misc, being replaced by: #'mrandersonicednrepl.orchard.v0v7v1.orchard.misc/update-vals
WARNING: update-keys already refers to: #'clojure.core/update-keys in namespace: mrandersonicednrepl.orchard.v0v7v1.orchard.misc, being replaced by: #'mrandersonicednrepl.orchard.v0v7v1.orchard.misc/update-keys
```

There is also a similar warning about medley, but I reported that separately in #21 , because it's a different dependency and the fix hasn't yet been released.

## Steps to Reproduce
1. In a `lein` project, update the Clojure dep to `1.11.0`
2. Run `iced repl`
3. Observe the warnings

## More Info
Orchard already has a fix released. I'll submit a PR here to bump the version.

**Comment by liquidz** [comment]
@stevejmp Just released iced-nrepl v1.2.9 and vim-iced v3.8.4 which uses the latest iced-nrepl https://github.com/liquidz/vim-iced/pull/394

Thank you so much!

**Comment by stevejmp** [comment]
こちらこそありがとうございます！
